src/lib.rs

```diff
@@ -359,14 +363,23 @@ pub trait PolynomialCommitment<G: AffineCurve>: Sized {
         )
     }
 
-    /// Build multi poly single point opening combination
+    /// Build the multi-poly single-point (random) linear combination. This is the standard reduction 
+    /// of a single-point multi-poly claim
+    /// `p_i(x) = v_i`, `i=1,...,m`, 
+    /// to a single-point single-poly claim on a random linear combination of the `p_i(X)`.
+    // TODO: the function should also compute the commitment of the linear combination.
+    // Of course not by doing an MSM, but doing the combination of the commitments as the verifier
+    // does.
     fn single_point_multi_poly_open_combine<'a>(
         labeled_polynomials: impl IntoIterator<Item = &'a LabeledPolynomial<G::ScalarField>>,
         labeled_commitments: impl IntoIterator<Item = &'a LabeledCommitment<G, Self::Commitment>>,
         fs_rng: &mut Self::RandomOracle,
         labeled_randomnesses: impl IntoIterator<Item = &'a LabeledRandomness<G, Self::Randomness>>,
```

Comment:
Same question on the need of the LabeledCommitment structure.

Reply:
Explained in the comment above